libevmasm/SemanticInformation.cpp

```diff
@@ -298,6 +298,13 @@ bool SemanticInformation::altersControlFlow(AssemblyItem const& _item)
 	}
 }
 
+bool SemanticInformation::terminatesControlFlow(AssemblyItem const& _item)
+{
+	if (_item.type() != evmasm::Operation)
+		return false;
+	return terminatesControlFlow(_item.instruction());
```

Comment:
We could additional apply the methods above if we hit a ``JUMP`` in the end, couldn't we? Not sure if that ever happens, though.

Reply:
I add a check for the `JUMP` as well.